wire-ios/Wire-iOS/Sources/UserInterface/ConversationList/Container/ConversationListViewController+NavigationBar/ConversationListViewController+NavigationBar.swift

```diff
@@ -412,7 +412,10 @@ extension ConversationListViewController: ConversationListContainerViewModelDele
     @objc
     private func presentConnectUI() {
         Task {
-            let connectUI = UINavigationController(rootViewController: connectViewControllerBuilder.build())
+            let featureConfigRepository = viewModel.userSession.clientSessionComponent?.featureConfigRepository
+            let isAppsTabVisible = await featureConfigRepository?.isFeatureEnabled(.apps) ?? false
```

Comment:
suggestion: move logic to viewModel
```suggestion
            let isAppsTabVisible = await viewModel.isAppsTabVisible
```

Reply:
I was able to delete those lines.